NEWS

```diff
@@ -1,21 +1,10 @@
-x/y/2015 ola-0.9.7
- Features:
- * 
-
- API:
- * 
-
- RDM Tests:
- * 
-
+12/7/2015 ola-0.9.7
```

Comment:
As soon as that. We need to get this resynced into master and gets the bots to run across it before we release.